config-gradle/action.yml

```diff
@@ -150,22 +167,34 @@ runs:
 
     # $GRADLE_USER_HOME is typically set to ~/.gradle/ by gradle/actions/setup-gradle
     - name: Configure Gradle Authentication
+      if: steps.config-gradle-completed.outputs.skip != 'true'
       shell: bash
       run: |
         GRADLE_INIT_DIR="$GRADLE_USER_HOME/init.d"
         mkdir -p "$GRADLE_INIT_DIR"
         cp "${GITHUB_ACTION_PATH}/resources/repoxAuth.init.gradle.kts" "$GRADLE_INIT_DIR/"
 
+    - name: Update project version and set current-version and project-version variables
+      id: set-version
+      if: steps.config-gradle-completed.outputs.skip != 'true' && (env.CURRENT_VERSION == '' || env.PROJECT_VERSION == '')
```

Comment:
This must be set, as it was implemented in Maven. See the other comment on build-maven.